src/cli.ts

```diff
@@ -49,120 +53,319 @@ const allLighthouseCategories = [
   'performance',
   'pwa',
   'seo',
-];
+] as const;
 
-sade('lighthouse-parade <url>', true)
+type RawFlags = Record<
+  string,
+  string | boolean | string[] | boolean[] | undefined
+> & { _: (string | number | boolean)[] };
+
+const cli = sade('lighthouse-parade [url]', true)
   .version(version)
   .example(
     'https://cloudfour.com --exclude-path-glob "/thinks/*" --max-crawl-depth 2 --output cloudfour-a.csv'
   )
   .describe(
     'Crawls the site at the provided URL, recording the lighthouse scores for each URL found.'
```

Comment:
Nit: Not related to this PR, but can we capitalize "Lighthouse" in this description? 🙂 
```suggestion
    'Crawls the site at the provided URL, recording the Lighthouse scores for each URL found.'
```